src/packet/route/link.rs

```diff
@@ -330,6 +334,47 @@ impl Links for NetlinkConnection {
         let reader = NetlinkReader::new(self);
         reader.read_to_end()
     }
+
+    fn link_set_down(&mut self, link: Link) -> io::Result<()> {
```

Comment:
If I'm not mistaken this is moving `Link` into here and then dropping it (same for set_up).